srcpkgs/Carla/template

```diff
@@ -10,19 +10,18 @@ makedepends="python3-PyQt5 libmagic file-devel libsndfile-devel
  liblo-devel alsa-lib-devel pulseaudio-devel libX11-devel gtk+3-devel
  gtk+-devel qt5-devel fluidsynth-devel fftw-devel zlib-devel
  python3-rdflib"
-depends="python3 python3-PyQt5 python3-PyQt5-svg which"
+depends="python3 python3-PyQt5 python3-PyQt5-svg pyliblo which"
 short_desc="Audio plugin host"
 maintainer="nutcase84 <nutcase84@protonmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://kxstudio.linuxaudio.org/Applications:Carla"
 distfiles="https://github.com/falkTX/Carla/archive/v${version}.tar.gz"
-checksum=a82ce08f3a82db9d878c8cb7e7e2f3b80834bf21801c6ec4ed95c0cfee25b963
+checksum=27def29cc408d5c74926e8d0ef3a77fd76fee1e4f2797f840e999e6376a5be03
 python_version=3
 
 case $XBPS_TARGET_MACHINE in
-	x86_64* | i686*);;
-	*) make_build_args+=" NOOPT=true"
-	   make_install_args+=" NOOPT=true";;
+	# No musl support for JUCE
+	*-musl) make_build_args+=" USING_JUCE=false";;
```

Comment:
What was the error?

Reply:
https://github.com/jkoderu-git/void-packages/runs/2478321591?check_suite_focus=true

Reply:
Interesting, it did work for `x86_64-musl`... So there's something wrong in the cross build for musl.

Reply:
I think I used another case when this PR was open first:
```
        x86_64 | i686);;
	x86_64-musl) make_build_args+=" USING_JUCE=false";;
```

I tested again without and it does not work for `x86_64-musl` either.
https://github.com/jkoderu-git/void-packages/pull/2

Reply:
Ok, errors happen here:

```
Compiling juce_audio_processors.cpp
In file included from ../../includes/vst3sdk/pluginterfaces/base/ftypes.h:19,
                 from ../../includes/vst3sdk/pluginterfaces/base/fstrdefs.h:19,
                 from ../../includes/vst3sdk/pluginterfaces/vst/vsttypes.h:19,
                 from format_types/juce_VST3Headers.h:102,
                 from format_types/juce_VST3PluginFormat.cpp:28,
                 from juce_audio_processors.cpp:152:
../../includes/vst3sdk/pluginterfaces/base/fplatform.h:188: warning: ignoring '#pragma error unknown' [-Wunknown-pragmas]
  188 |  #pragma error unknown platform
      | 
In file included from format_types/juce_VST3Headers.h:109,
                 from format_types/juce_VST3PluginFormat.cpp:28,
                 from juce_audio_processors.cpp:152:
../../includes/vst3sdk/base/source/fstring.cpp:1998:2: warning: #warning DEPRECATED No Linux implementation [-Wcpp]
 1998 | #warning DEPRECATED No Linux implementation
      |  ^~~~~~~
In file included from format_types/juce_VST3Headers.h:112,
                 from format_types/juce_VST3PluginFormat.cpp:28,
                 from juce_audio_processors.cpp:152:
../../includes/vst3sdk/base/thread/source/flock.cpp:81:2: warning: #warning implement FLock! [-Wcpp]
   81 | #warning implement FLock!
      |  ^~~~~~~
In file included from format_types/juce_VST3Headers.h:120,
                 from format_types/juce_VST3PluginFormat.cpp:28,
                 from juce_audio_processors.cpp:152:
../../includes/vst3sdk/pluginterfaces/base/funknown.cpp:78:2: warning: #warning implement me! [-Wcpp]
   78 | #warning implement me!
      |  ^~~~~~~
In file included from format_types/juce_VST3Headers.h:122,
                 from format_types/juce_VST3PluginFormat.cpp:28,
                 from juce_audio_processors.cpp:152:
../../includes/vst3sdk/pluginterfaces/base/ustring.cpp:163:2: warning: #warning Implement me [-Wcpp]
  163 | #warning Implement me
      |  ^~~~~~~
../../includes/vst3sdk/pluginterfaces/base/ustring.cpp:201:2: warning: #warning Implement me [-Wcpp]
  201 | #warning Implement me
      |  ^~~~~~~
../../includes/vst3sdk/pluginterfaces/base/ustring.cpp:228:2: warning: #warning Implement me [-Wcpp]
  228 | #warning Implement me
      |  ^~~~~~~
../../includes/vst3sdk/pluginterfaces/base/ustring.cpp:267:2: warning: #warning Implement me [-Wcpp]
  267 | #warning Implement me
      |  ^~~~~~~
Creating juce_audio_basics.a
make[1]: Leaving directory '/builddir/Carla-2.3.0/source/modules/juce_audio_basics'
make[1]: Entering directory '/builddir/Carla-2.3.0/source/modules/juce_core'
Compiling juce_core.cpp
In file included from juce_core.cpp:69:
/usr/aarch64-linux-musl/usr/include/sys/errno.h:1:2: warning: #warning redirecting incorrect #include <sys/errno.h> to <errno.h> [-Wcpp]
    1 | #warning redirecting incorrect #include <sys/errno.h> to <errno.h>
      |  ^~~~~~~
juce_core.cpp:95:12: fatal error: execinfo.h: No such file or directory
   95 |   #include <execinfo.h>
      |            ^~~~~~~~~~~~
compilation terminated.
```

some of the macros weren't enough.

Reply:
I think we can consider merging as is if you add a big `FIXME` comment here about `juce` on musl, but we might be able to fix...

Reply:
Maybe it's due to being run in your fork? Feel free to push it here for CI to run locally :)

Reply:
> I changed the case in my repo to try a fix for musl
> 
> ```
> if [ "$XBPS_TARGET_LIBC" = musl ]; then
> 	makedepends+=" libexecinfo-devel"
> fi
> ```
> 
> but I can't test because of a CI error.
> 
> ```
> Run common/travis/changed_templates.sh
> Changed packages:
> fatal: bad revision ''
> ```

It does not work. I add the `FIXME` then.